bridge/qb/server/functions.lua

```diff
@@ -37,6 +37,7 @@ function functions.SpawnVehicle(source, model, coords, warp)
     local veh = CreateVehicle(model, coords.x, coords.y, coords.z, heading, true, true)
     while not DoesEntityExist(veh) do Wait(0) end
     if warp then
+        ---@diagnostic disable-next-line: missing-parameter
         while GetVehiclePedIsIn(ped) ~= veh do
```

Comment:
Can just add the missing parameter?

Reply:
Seeing as this is in the bridge, I opted to add the diagnostic instead of adding the parameter as qb-core has it just like this. If we think it'd be better to add the `lastVehicle` boolean instead, we can.

Reply:
I assume a lack of a parameter here is the same as `false`? Seems to also be the intention